libnetwork/iptables/conntrack.go

```diff
@@ -102,5 +103,6 @@ func purgeConntrackState(nlh *netlink.Handle, family netlink.InetFamily, ipAddre
 	if err := filter.AddIP(netlink.ConntrackNatAnyIP, ipAddress); err != nil {
 		return 0, err
 	}
-	return nlh.ConntrackDeleteFilters(netlink.ConntrackTable, family, filter)
+	nDeleted, err := nlutil.ConntrackDeleteFilters(nlh, netlink.ConntrackTable, family, filter)
+	return nDeleted, err
```

Comment:
These intermediate vars aren't used for anything, we can remove them.

```suggestion
	return nlutil.ConntrackDeleteFilters(nlh, netlink.ConntrackTable, family, filter)
```

Reply:
Oh, weird, thank you - not sure what happened there, I must have been experimenting with something!